pylutron/__init__.py

```diff
@@ -294,8 +325,12 @@ def _parse_output(self, output_xml):
 
   def _parse_keypad(self, keypad_xml, device_group):
     """Parses a keypad device (the Visor receiver is technically a keypad too)."""
+    # keypad standard name is CSD 001, we use the integration ID nme instead
+    name = keypad_xml.get('Name') 
+    if (keypad_xml.get('Name') == "CSD 001"):
+      name = f"keypad {keypad_xml.get('IntegrationID')}"
```

Comment:
I'm using keypad + integrationID because integrationID is unique. Keypad is not. I've multiple keypads in the same room. In Hass the name is `return f"{self._area_name} {self._lutron_device.name}"`, so two keypads in the same area would have the same name

Reply:
This is something to handle in Home Assistant.  As long as these have UUIDs, then home assistant can handle 2 keypads with the same name.

Reply:
We don't have UUIDs in our XML

Reply:
Can you post some sample XML in the PR?  A lot has gone into using UUIDs in the home assistant integration and we should make sure it's not going to cause problems.

In theory, as long as everything has uniquie integration IDs, the fallback mechanism of using integration IDs should still result in unique devices and entities that can be named/renamed as you'd like.  But if someone configures their setup to have 5 devices all named keypad in the same room, who are we to tell them they should be unique names?